plugins/ros/src/components/riScInfo/riScStatus/RiScStatusComponent.tsx

```diff
@@ -110,71 +129,130 @@ export const RiScStatusComponent = ({
 
   const [status, setStatus] = useState<0 | 1 | 2 | 3>(0);
 
+  const STATUS_CREATED = 0;
+  const STATUS_DRAFT = 1;
+  const STATUS_WAITING = 2;
+  const STATUS_PUBLISHED = 3;
+
   useEffect(() => {
     if (selectedRiSc.content.scenarios.length === 0) {
-      setStatus(0);
+      setStatus(STATUS_CREATED);
     } else if (selectedRiSc.status === RiScStatus.Draft) {
-      setStatus(1);
+      setStatus(STATUS_DRAFT);
     } else if (selectedRiSc.status === RiScStatus.SentForApproval) {
-      setStatus(2);
+      setStatus(STATUS_WAITING);
     } else if (selectedRiSc.status === RiScStatus.Published) {
-      setStatus(3);
+      setStatus(STATUS_PUBLISHED);
     }
   }, [selectedRiSc.status, selectedRiSc.content.scenarios]);
 
   const migration = selectedRiSc.migrationStatus?.migrationChanges;
 
+  const lastModifiedDate = selectedRiSc.sopsConfig.lastModified
+    ? new Date(selectedRiSc.sopsConfig.lastModified).toLocaleDateString(
+        'no-NO',
+        {
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric',
+        },
+      )
+    : null;
+
+  const dateString = selectedRiSc.sopsConfig.lastModified;
+
+  const daysSinceLastModified = dateString
+    ? calculateDaysSinceLastModified(dateString)
+    : null;
+
+  const numOfCommitsBehind = selectedRiSc.numOfGeneralCommitsBehind
+    ? selectedRiSc.numOfGeneralCommitsBehind.toString()
+    : null;
+
+  const updatedState = useMemo(() => {
+    if (daysSinceLastModified && numOfCommitsBehind) {
+      const days = parseInt(daysSinceLastModified);
+      const commits = parseInt(numOfCommitsBehind);
+
+      if (commits > 50) {
+        return 'very_outdated';
+      } else if (commits >= 26 && commits <= 50) {
```

Comment:
This might just be down to preference, but it would be easier to read this statement (and others below) if the order of the arguments in the first condition are interchanged, i.e.,
```suggestion
      } else if (26 <= commits && commits <= 50) {
```

Reply:
syns egentlig det gir mening med det andre bare fordi da vet du **hva** du sammenligner med før du får vite tallet?

Reply:
Er i stor grad preferanse ja, jeg har tilfeldigvis samme som Halvard.
For å gi et annet perspektiv på det, så skulle man ønske at syntaxen støttet
`n <= commits <= x`
Det fungerer ikke, så forslaget her er det nærmeste man kommer. Man ser basically på helheten i stedet for to separate sjekker